src/lib/Libwin/accesinfo.c

```diff
@@ -184,22 +194,43 @@ secure_file(char *path, char *user, ACCESS_MASK mask)
 	char	logb[LOG_BUF_SIZE] = {'\0' } ;
 	char	*gname = NULL;
 
-	if (path == NULL || *path == '\0')
+	if (path == NULL || *path == '\0') {
+		sprintf(logb, "Invalid path");
+		log_err(-1, __func__, logb);
```

Comment:
errno?

Reply:
I didn't understand. If the path is NULL then what error code we are expecting ? I mean we are not calling in API here

Reply:
yeah ignore that, no api call there